build.sbt

```diff
@@ -332,16 +315,8 @@ lazy val jmh: Project = project
       "com.google.apis" % "google-api-services-bigquery" % bigqueryVersion % Test,
       "joda-time" % "joda-time" % jodaTimeVersion % Test,
       "com.google.cloud.datastore" % "datastore-v1-proto-client" % datastoreVersion % Test,
-      "com.google.api.grpc" % "proto-google-common-protos" % grpcApiVersion % Test,
       "org.tensorflow" % "proto" % tensorflowVersion % Test
-    ),
-    // proto config
-    protobufProtocOptions in ProtobufConfig ++= Seq("--include_std_types"),
```

Comment:
What are these flags for? Works fine without them.

Reply:
saw it in another example, fine with me to remove